src/common/index.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import Handsontable from 'hot-alias';
-import {SettingsMapper} from './settingsMapper';
+import { SettingsMapper } from './settingsMapper';
+
+export interface Props extends Handsontable.DefaultSettings {
```

Comment:
I think it should be separated file. Would you mind to add a description? What it is at all.

Reply:
By the way, maybe this interface should be named as `HotTableProps`?